packages/shared/src/hooks/useBookmarkPost.ts

```diff
@@ -43,7 +43,7 @@ export default function useBookmarkPost<
     {
       onMutate: onBookmarkMutate,
       onError: (err, _, rollback) => rollback?.(),
-      onSuccess: () => trackEvent(onBookmarkTrackObject()),
+      onSuccess: () => trackEvent(onBookmarkTrackObject?.()),
```

Comment:
If `onBookmarkTrackObject` is not defined we should not call `trackEvent`

```suggestion
      onSuccess: () => onBookmarkTrackObject && trackEvent(onBookmarkTrackObject()),
```

Reply:
@idoshamun done 👍